digiwf-libs/digiwf-message/README.md

```diff
@@ -5,24 +5,22 @@ reduces technical complexity.
 
 The idea behind the **DigiWF Message** library is to solve the recurring Spring Cloud Stream configurations in a central
 location and provide an API to use them.
-This way, an integration developer no longer needs to delve into the depths of Spring Cloud Stream configurations and
+This way, an integration developer no longer needs to dive into the depths of Spring Cloud Stream configurations and
 can focus on implementing the integration.
 
-The library provides its own APIs for sending messages that send messages to a message broker.
-In addition, the library also provides a Spring Cloud Stream `RoutingCallback` for consuming and forwarding messages (
-function routing).
+The library provides its own APIs for sending messages to a message broker.
+Additionally, the library also configures event routing, where messages can be assigned to a `Consumer` simply by
+matching the name of the header type and the consumer.
 
 ## Usage
 
 The **DigiWF Message** library provides the MessageApi, which is used to send messages.
 
-In addition, APIs are provided for recurring messages that build on the MessageApi.
+In addition, APIs are provided for recurring messages to the corresponding destinations that build on the MessageApi.
```

Comment:
Finde den ganzen Satz irgendwie nicht so geil, aber mir ist keine bessere Formulierung eingefallen.
Meiner Meinung nach geht es auch um `recurring message types`.

Reply:
`In addition, APIs are provided for recurring message types that build on the MessageApi.`

So ist der Satz noch ein wenig informativ und nicht mehr falsch.